bin/src/command/mod.rs

```diff
@@ -627,11 +627,12 @@ pub fn start(
         })
         .detach();
 
-        let saved_state = config.saved_state_path();
+        let saved_state_path = config.saved_state.clone();
```

Comment:
This modification seems to change the behaviour of the function. Could you explain me the reason why you update this part?

Reply:
The whole deal is that calling `config.saved_state_path()` does not allow error propagation in this place (a detached thread).

This PR puts the call of `config.saved_state_path()` much earlier in Sōzu startup, by editing `config.saved_state` upstream.